SeleniumTask/src/test/java/GitHubPage.java

```diff
@@ -0,0 +1,17 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static constants.Constants.LOGIN_XPATH;
+
+class GitHubPage {
```

Comment:
move page objects from test folder to src
in test folder should be only test cases